### ISSUE TITLE
Error in The Form Component

### ISSUE THREAD

**Issue by topwebstudio** [symfony/symfony-docs#7242]
https://symfony.com/doc/master/components/form.html

```
$twig->addExtension(
    new FormExtension(new TwigRenderer($formEngine, $csrfManager))
);
```

This line throws an error.

> Uncaught exception 'InvalidArgumentException' with message 'Passing any arguments the constructor of Symfony\Bridge\Twig\Extension\FormExtension is reserved for internal use.'

**Comment by xabbuh** [comment]
We indeed need to update the example. Thank you for reporting it @topwebstudio!

However, I wonder why you get the exception and do not see a deprecation being triggered here: https://github.com/symfony/symfony/blob/3.2/src/Symfony/Bridge/Twig/Extension/FormExtension.php#L35

**Comment by topwebstudio** [comment]
I followed the guide for the Form component outside Symfony scope (as a standalone).

**Comment by xabbuh** [comment]
@topwebstudio Indeed, I was able to reproduce your initial issue (see the related PR symfony/symfony#20882 which will lead to the expected deprecation instead of the exception).

Nonetheless, we still need to update the documentation. So thanks again for reporting the issue! :)

**Comment by topwebstudio** [comment]
I don't know if this is related to Form Component documentation errors above, but when following the official documentation I'm running into new error:

> Fatal error: Uncaught exception 'Twig_Error_Runtime' with message 'Unable to load the "Symfony\Bridge\Twig\Form\TwigRenderer" runtime.'

I've made pretty minimalistic example to display this error: 
https://github.com/topwebstudio/symfonyform

**Comment by xabbuh** [comment]
@topwebstudio Can you report this as a bug in the code repository?

**Comment by topwebstudio** [comment]
Here is the bug report [link](https://github.com/symfony/symfony/issues/21008)

**Comment by Mohammadtrabelsi** [comment]
I should open this issue in sonata paga bundle repo
This error is fixed by upgrading sonata page bundle to 3.7.1
